src/main/java/se/liu/ida/hefquin/engine/wrappers/graphqlwrapper/data/impl/GraphQLEntrypointImpl.java

```diff
@@ -9,12 +9,14 @@ public class GraphQLEntrypointImpl implements GraphQLEntrypoint {
     protected final String fieldName;
     protected final Map<String,String> argumentDefinitions;
     protected final String type;
+    protected final GraphQLEntrypointType epType;
 
     public GraphQLEntrypointImpl(final String fieldName, final Map<String,String> argumentDefinitions,
-            final String type){
+            final String type, final GraphQLEntrypointType epType){
```

Comment:
the "type" string refers to the name of what type of objects the entrypoint fetches (i.e. the graphql types) so the names it can have isn't pre-determined. The epType is simply an enum that describes whether the entrypoint fetches a single object, a filtered list of objects or a full list of objects.